libraries/AP_Param/AP_Param.cpp

```diff
@@ -342,7 +342,6 @@ bool AP_Param::setup(void)
             hdr2.revision == k_EEPROM_revision &&
             _storage.copy_area(_storage_bak)) {
             // restored from backup
-            INTERNAL_ERROR(AP_InternalError::error_t::params_restored);
```

Comment:
```suggestion
            AP::logger().Write_Event(LogEvent::PARAMS_RESTORED);
```

... and add to enumeration in `AP_Logger/AP_Logger.h`

Reply:
AP_Logger isn't even initialised when this happens. Param load happens first thing on startup